packages/app/package.json

```diff
@@ -116,6 +117,7 @@
     "eventemitter2": "^6.4.9",
     "extension-port-stream": "^2.1.1",
     "fast-deep-equal": "^3.1.3",
+    "json-stable-stringify": "^1.0.2",
```

Comment:
Actually, I don't trust such packages and relying on string representation of object can be danger due to compatibility and collisions. I think we can use https://docs.ethers.org/v5/api/utils/hashing/#TypedDataEncoder but it's still in experimental stage so if there is another option, let's discuss.

Reply:
Ok, yes I couldn't find any other options, as you mentioned the one you linked is experimental. Will explore other options but the package I found seemed to be the most widely used/stable